iam/api-client/service_accounts_test.py

```diff
@@ -22,7 +22,7 @@
 
 def test_service_accounts(capsys):
     project_id = os.environ['GOOGLE_CLOUD_PROJECT']
-    name = f'test-{uuid.uuid4().hex[:25]}'
+    name = "python-test-" + str(uuid.uuid4()).split('-')[0]
```

Comment:
f-string is recommended for newer python.

Reply:
Fixed